### DISCUSSION TITLE
Official 7" Screen Not Working with RPi4

### DISCUSSION THREAD

**Discussion by feebb** [rern/rAudio#280] [Fixed Issues]
I can't get the  Raspberry Pi Official 7" Screen to work with the latest version of rAudio on a Raspberry Pi 4. Tried rebooting, different RPi4s, another of the same screen, turning on Browser on RPi in Features, refreshing--the screen only shows the rAudio symbol, flashes for a second, and then goes black.

I get the following when I click on Browser on RPi in Features:

` pacman -Q chromium; systemctl status localbrowser

chromium 89.0.4389.82-2
● localbrowser.service - Local Browser - Chromium
     Loaded: loaded (/etc/systemd/system/localbrowser.service; enabled; vendor preset: disabled)
     Active: failed (Result: exit-code) since Wed 2021-03-31 11:53:06 EDT; 5s ago
    Process: 909 ExecStart=/usr/bin/sudo /usr/bin/startx (code=exited, status=0/SUCCESS)
    Process: 1081 ExecStop=/usr/bin/killall Xorg (code=exited, status=1/FAILURE)
   Main PID: 909 (code=exited, status=0/SUCCESS)`

The screen **does** work with the latest version of rAudio with a RPi 3B+. On the 3B, the screen flashes white and then displays everything normally.

**Comment by ceejayemm** [comment]
Have you by chance changed the Host Name (System /  Environment) ?  I have had your problem (on a RPi 3B+, rAudio 1 • 20210330 and official screen) when I have done this and the problem is caused by the Chromium profile being locked to the original hostname. To correct this I do:

cd ~/.config/chromium
mv SingletonCookie CM_SingletonCookie
mv SingletonLock CM_SingletonLock			<-- Most likely culprit caused by change of hostname
mv SingletonSocket CM_SingletonSocket
systemctl restart localbrowser
systemctl status localbrowser

This usually solves this problems for me.

Chris

**Reply by feebb** [reply]
Thanks very much, Chris! That was the problem and it is now fixed!
